scrapi/base/__init__.py

```diff
@@ -110,26 +136,29 @@ def get_contributors(self, result):
         or creators in the OAI namespaces """
 
         contributors = result.xpath(
-            '//dc:contributor/node()', namespaces=self.NAMESPACES) or ['']
```

Comment:
`['']` is a list with a singular empty string in it where `[]` is just an empty list.
Was this change intended?